main.js

```diff
@@ -1,8 +1,59 @@
 const Apify = require('apify');
-const getProjects = require('./src/getProjects');
+const querystring = require('querystring');
+
+const { parseInput } = require('./src/utils');
+const { BASE_URL, PROJECTS_PER_PAGE } = require('./src/consts');
+const { handleStart, handlePagination } = require('./src/routes');
+
+const { log } = Apify.utils;
 
 Apify.main(async () => {
-    // Load query from input
-    const input = await Apify.getValue('INPUT');
-    await getProjects(input);
+    const proxyConfiguration = await Apify.createProxyConfiguration();
```

Comment:
use the proxy helper from https://github.com/metalwarrior665/apify-utils/blob/master/src/validation.js#L23

people with expired trials will try to run using GOOGLESERP group most of the time
place the proxy configuration on the INPUT_SCHEMA.json please